app/src/main/res/layout/act_profile.xml

```diff
@@ -246,13 +246,32 @@
             android:layout_height="0dp"
             android:layout_weight="1"/>
 
-        <ImageButton
-            android:id="@+id/profile_Edit"
+        <LinearLayout
             android:layout_width="match_parent"
-            android:layout_height="43dp"
-            android:layout_marginTop="4dp"
-            android:contentDescription="@string/edit"
-            app:srcCompat="@drawable/ic_baseline_edit_24" />
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <ImageButton
+                android:id="@+id/profile_Logout"
+                android:layout_width="match_parent"
+                android:layout_height="43dp"
+                android:layout_weight="1"
+                android:layout_marginTop="4dp"
+                android:contentDescription="@string/edit"
+                app:srcCompat="@drawable/ic_logout" />
+
+            <ImageButton
+                android:id="@+id/profile_Edit"
+                android:layout_width="match_parent"
+                android:layout_height="43dp"
+                android:layout_weight="1"
+                android:layout_marginTop="4dp"
+                android:contentDescription="@string/edit"
+                app:srcCompat="@drawable/ic_edit" />
```

Comment:
Not sure if the issue can be my emulator, but the buttons don't show here.  This is weird because the other activities inflate above the bottom menu, but this one overlaps.